lua/telescope/previewers/buffer_previewer.lua

```diff
@@ -1029,44 +1029,49 @@ previewers.highlights = defaulter(function(_)
       return "highlights"
     end,
 
-    define_preview = function(self, entry, status)
-      putils.with_preview_window(status, nil, function()
-        if not self.state.bufname then
-          local output = vim.split(vim.fn.execute "highlight", "\n")
-          local hl_groups = {}
-          for _, v in ipairs(output) do
-            if v ~= "" then
-              if v:sub(1, 1) == " " then
-                local part_of_old = v:match "%s+(.*)"
-                hl_groups[#hl_groups] = hl_groups[#hl_groups] .. part_of_old
-              else
-                table.insert(hl_groups, v)
-              end
+    define_preview = function(self, entry)
+      if not self.state.bufname then
+        local output = vim.split(vim.fn.execute "highlight", "\n")
+        local hl_groups = {}
+        for _, v in ipairs(output) do
+          if v ~= "" then
+            if v:sub(1, 1) == " " then
+              local part_of_old = v:match "%s+(.*)"
+              hl_groups[#hl_groups] = hl_groups[#hl_groups] .. part_of_old
+            else
+              table.insert(hl_groups, v)
             end
           end
+        end
 
-          vim.api.nvim_buf_set_lines(self.state.bufnr, 0, -1, false, hl_groups)
-          for k, v in ipairs(hl_groups) do
-            local startPos = string.find(v, "xxx", 1, true) - 1
-            local endPos = startPos + 3
-            local hlgroup = string.match(v, "([^ ]*)%s+.*")
-            pcall(vim.api.nvim_buf_add_highlight, self.state.bufnr, 0, hlgroup, k - 1, startPos, endPos)
-          end
+        vim.api.nvim_buf_set_lines(self.state.bufnr, 0, -1, false, hl_groups)
+        for k, v in ipairs(hl_groups) do
+          local startPos = string.find(v, "xxx", 1, true) - 1
+          local endPos = startPos + 3
+          local hlgroup = string.match(v, "([^ ]*)%s+.*")
+          pcall(vim.api.nvim_buf_add_highlight, self.state.bufnr, 0, hlgroup, k - 1, startPos, endPos)
         end
+      end
+      print(self.state.bufnr)
```

Comment:
looks like you left a print statement here

Reply:
thx :)